Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/JaxbXmlUtils.java

```diff
@@ -53,7 +55,9 @@ private JaxbXmlUtils() {
      * @return The Result of the transformation as String object.
      */
     static String transformXmlByXslt(URI xmlFile, URI xslFile) throws TransformerException, IOException {
-        StreamSource source = new StreamSource(xmlFile.getPath());
+        FileManagementInterface fileManagementModule = new KitodoServiceLoader<FileManagementInterface>(
+                FileManagementInterface.class).loadModule();
+        StreamSource source = new StreamSource(fileManagementModule.getFile(xmlFile).toURI().toString());
```

Comment:
FileManagement should be used to open the InputStream. The URI may have a schema understood only by FileManagement (that is, the data is in the repository). Maybe like this (I have reordered the lines so you can do with just one try-with-resources statement):

    TransformerFactory factory = TransformerFactory.newInstance();
    StreamSource styleSource = new StreamSource(xslFile.getPath());
    Transformer transformer = factory.newTransformer(styleSource);
    try (InputStream inputStream = fileManagementModule.read(xmlFile);
            StringWriter stringWriter = new StringWriter()) {
        StreamSource source = new StreamSource(inputStream);
        StreamResult result = new StreamResult(stringWriter);
        transformer.transform(source, result);
        return stringWriter.toString();
    }